modules/data-loading/pages/s3-loader-user-guide.adoc

```diff
@@ -63,75 +52,65 @@ This SET command reads, validates, and applies the configuration file, integrati
 }
 ----
 
-
-
-For simplicity, you can merge the CREATE DATA_SOURCE and SET statements:
+You can also provide the path to the data source variable at creation instead of using the `SET` command to specify the path:
 
 [,gsql]
 ----
-CREATE DATA_SOURCE S3 data_source_name = "/path/to/s3.config"
+CREATE DATA_SOURCE S3 example_data_source = "/path/to/s3.config"
 ----
 
-[NOTE]
-====
 
-. If you have a TigerGraph cluster, the configuration file must be on machine m1, where the GSQL server and GSQL client both reside,  and it must be in JSON format. If the configuration file uses a relative path, the path should be relative to the GSQL client working directory.
-. Each time when the config file is updated, you must run "SET data_source_name"  to update the data source details in the dictionary.
-====
+If you have a TigerGraph cluster, the configuration file must be on machine m1, where the GSQL server and GSQL client both reside,  and it must be in JSON format.
+If the configuration file uses a relative path, the path should be relative to the GSQL client working directory.
 
-To further simplify, instead of specifying the S3 data source config file path, you can also directly provide the S3 data source configuration as a string argument, as shown below:
+Each time when the config file is updated, you must run `SET data_source_name`  to update the data source details.
 
-[,gsql]
+To further simplify, instead of specifying the S3 data source config file path, you can also directly provide the S3 data source configuration as a string argument, as shown below.
+This is useful when you only have access to the GSQL shell, but do not have OS level access such as when you are on TigerGraph Cloud:
+
+[.wrap,gsql]
 ----
 CREATE DATA_SOURCE S3 data_source_name = "{\"file.reader.settings.fs.s3a.access.key\":\"AKIAJ****4YGHQ\",\"file.reader.settings.fs.s3a.secret.key\":\"R8bli****p+dT4\"}"
 ----
 
-[NOTE]
-====
-*Tip*: The above simplified statement is useful for using S3 Data Loader in TigerGraph Cloud when you are not loading S3 data through GraphStudio. In TigerGraph Cloud (tgcloud.io), you can also use GSQL web shell to define and create S3 data sources, without creating the S3 data source configuration file in filesystem.
-====
-
-==== ADVANCED: MultiGraph Support
+==== Global and local data sources
 
 The S3 Loader supports the TigerGraph MultiGraph feature. In the MultiGraph context, a data source can be either global or local:
 
-. A global data source can only be created by a superuser, who can grant the global data source to any graph.
-. An admin user can only create a local data source, which cannot be accessed by other graphs.
+* A global data source can only be created by a user with the `WRITE_DATASOURCE` privilege on the global scope, who can grant the global data source to any graph.
+* An user with `WRITE_DATASOURCE` privilege on a graph can only create local data sources on that graph, which cannot be accessed by other graphs.
```

Comment:
a user with

Consistency in "with the X privilege" vs "with X privilege"

I think the second sentence sounds like the graph cannot be accessed by other graphs. Maybe break it into two sentences?